lib/Dialect/Calyx/Export/CalyxEmitter.cpp

```diff
@@ -899,6 +907,20 @@ void Emitter::emitInvoke(InvokeOp invoke) {
   os << RParen() << semicolonEndL();
 }
 
+void Emitter::emitConstant(ConstantOp constantOp) {
+  TypedAttr attr = constantOp.getValueAttr();
+  assert(isa<FloatAttr>(attr) && "must be a floating point constant");
+  auto fltAttr = cast<FloatAttr>(attr);
+  APFloat value = fltAttr.getValue();
+  auto type = cast<FloatType>(fltAttr.getType());
+  double doubleValue = value.convertToDouble();
+  auto floatBits = value.getSizeInBits(type.getFloatSemantics());
+  indent() << constantOp.getName().str() << space() << equals() << space()
+           << "std_float_const";
+  os << LParen() << "0" /* IEEE754 */ << comma() << floatBits << comma()
```

Comment:
Perhaps spell out this magic number a bit more explicitly, something like:
```suggestion
  // Currently defaults to IEEE-754 representation [1]. 
  // [1]: https://github.com/calyxir/calyx/blob/main/primitives/float.futil
  static constexpr int32_t IEEE754 = 0;
  os << LParen() << std::to_string(IEEE754) << comma() << floatBits << comma()
```